api/src/main/java/io/strimzi/api/kafka/model/CertificateAuthority.java

```diff
@@ -58,6 +59,17 @@ public void setGenerateCertificateAuthority(boolean generateCertificateAuthority
         this.generateCertificateAuthority = generateCertificateAuthority;
     }
 
+    @Description("If true then the Certificate Authority certificates secrets owner reference is set to the Kafka object. " +
+    "Otherwise no owner reference is set " +
+    "Default is true.")
```

Comment:
should we explain what's the consequence of setting it to false? so explaining that deleting the Kafka cluster won't delete the certificate secrets?